osu.Game.Tests/Visual/UserInterface/TestSceneCommentEditor.cs

```diff
@@ -108,7 +107,7 @@ private partial class TestCommentEditor : CommentEditor
             public new Bindable<string> Current => base.Current;
             public new FillFlowContainer ButtonsContainer => base.ButtonsContainer;
 
-            public string CommittedText { get; private set; }
+            public string? CommittedText { get; private set; }
```

Comment:
Generally we prefer using `= string.Empty` instead of string nullability if possible.